README.md

```diff
@@ -24,3 +28,8 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+
+Running the tests
+
+To run all tests, run `pytest`
```

Comment:
It's only `pytest` if from a poetry venv shell. Otherwise it needs to be `poetry run pytest`